13_sandboxes/jupyter_sandbox.py

```diff
@@ -62,14 +64,15 @@
         timeout=5 * 60,  # 5 minutes
         image=image,
         app=app,
+        gpu=None,  # add a GPU if you need it!
     )
 
 print(f"🏖️  Sandbox ID: {sandbox.object_id}")
 
 # ## Communicating with a Jupyter server
 
 # Next, we print out a URL that we can use to connect to our Jupyter server.
-# Note that we have to call [`Sandbox.tunnels`](/docs/reference/modal.Sandbox#tunnels)
+# Note that we have to call [`Sandbox.tunnels`](https://modal.com/docs/reference/modal.Sandbox#tunnels)
```

Comment:
Which dev workflows does it break?

I use explicit/absolute URLs because the examples repo code files are meant to stand on their own. So for example we want the links to be clickable from inside the IDE.

Reply:
Ah I see what you mean. Running locally or in our dev cluster, the relative link will now redirect to Modal instead of the same host it was started on. But I think what you have makes more sense for people running the examples themselves.

Reply:
Ah I see -- it slows you down when simultaneously developing several linked pages.

But I think that's a tradeoff we end up with because our examples are public but other docs are private.

Reply:
yeah u right